packages/runtime/src/templates/getPageResolver.ts

```diff
@@ -1,30 +1,43 @@
 import { posix } from 'path'
 
+import glob from 'globby'
 import { outdent } from 'outdent'
 import { relative, resolve } from 'pathe'
 import slash from 'slash'
-import glob from 'tiny-glob'
 
 import { HANDLER_FUNCTION_NAME } from '../constants'
 import { getDependenciesOfFile } from '../helpers/files'
 
 // Generate a file full of require.resolve() calls for all the pages in the
 // build. This is used by the nft bundler to find all the pages.
 
-export const getPageResolver = async ({ publish, target }: { publish: string; target: string }) => {
-  const functionDir = posix.resolve(posix.join('.netlify', 'functions', HANDLER_FUNCTION_NAME))
-  const root = posix.resolve(slash(publish), target === 'server' ? 'server' : 'serverless', 'pages')
+export const getUniqueDependencies = async (sourceFiles: Array<string>) => {
+  const dependencies = await Promise.all(sourceFiles.map((sourceFile) => getDependenciesOfFile(sourceFile)))
+  return [...new Set([...sourceFiles, ...dependencies.flat()])].sort()
+}
 
-  const pages = await glob('**/*.js', {
+export const getAllPageDependencies = async (publish: string) => {
+  const root = posix.resolve(slash(publish), 'server')
```

Comment:
I know it's only `'server'` now, but I can't remember the reason why.

Reply:
We force the target to be server now